scripts/publishing-common.sh

```diff
@@ -13,7 +13,8 @@
 # License for the specific language governing permissions and
 # limitations under the License.
 
-export VERSION=$(cat $(dirname "${0}")/../VERSION)
+DIR="$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )"
+export VERSION=$(cat ${DIR}/../VERSION)
```

Comment:
Please "quote" the path given to cat.
```suggestion
export VERSION=$(cat "${DIR}/../VERSION")
```